flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/JarHandlerParameterTest.java

```diff
@@ -211,6 +218,37 @@ void testProvideJobId() throws Exception {
         assertThat(jobGraph.get().getJobID()).isEqualTo(jobId);
     }
 
+    @Test
+    void testProvideFlinkConfig() throws Exception {
+
+        REQB jarRequestBodyWithFlinkConfig = getJarRequestBodyWithFlinkConfig(FLINK_CONFIGURATION);
+        HandlerRequest<REQB> request =
+                createRequest(
+                        getJarRequestBodyWithFlinkConfig(FLINK_CONFIGURATION),
+                        getUnresolvedJarMessageParameters(),
+                        getUnresolvedJarMessageParameters(),
+                        jarWithManifest);
+
+        handleRequest(request);
+
+        Optional<JobGraph> jobGraph = getLastSubmittedJobGraphAndReset();
+
+        assertThat(jobGraph.isPresent()).isTrue();
```

Comment:
This isn't the case of the JarPlanHandler.

Reply:
JarPlanHandler not need to change,if I just want to test JarRunPlaner flinkConfig is ready, should I test in the test method?

Reply:
Now I've added the logic of the JobPlanHandler part

Reply:
Now I've added the logic of the JobPlanHandler part,not need to case it